.github/workflows/formatting.yml

```diff
@@ -0,0 +1,50 @@
+name: Formatting
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/pr-fetch@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download stanc3
+        run: |
+          cd ..
+          wget https://github.com/stan-dev/stanc3/releases/download/nightly/linux-stanc
```

Comment:
For now, this uses the nightly release because that one has .stanfunctions support. Then we should switch to using release binaries.